.github/workflows/linting.yml

```diff
@@ -11,72 +11,33 @@ on:
     types: [published]
 
 jobs:
-  EditorConfig:
+  pre-commit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
-      - uses: actions/setup-node@v3
-
-      - name: Install editorconfig-checker
-        run: npm install -g editorconfig-checker
-
-      - name: Run ECLint check
-        run: editorconfig-checker -exclude README.md $(find .* -type f | grep -v '.git\|.py\|.md\|json\|yml\|yaml\|html\|css\|work\|.nextflow\|build\|nf_core.egg-info\|log.txt\|Makefile')
-
-  Prettier:
```

Comment:
Thanks. Does this mean that other nf-core pipelines also no longer run `prettier` or `editorconfig-checker` (and so why we are removing these here)?

Reply:
Yeah, so you can see the template file (save for some dynamic substitutions that get made) for 2.12.1 here: https://github.com/nf-core/tools/blob/2.12.1/nf_core/pipeline-template/.github/workflows/linting.yml

They've seemingly removed prettier and also replaced black for ruff.

Reply:
Okay. Thank so much.